compiler/modules/nback/number.bal

```diff
@@ -95,6 +109,21 @@ function buildFloatArithmeticBinary(llvm:Builder builder, Scaffold scaffold, bir
     buildStoreFloat(builder, scaffold, result, insn.result);                                  
 }
 
+function buildDecimalArithmeticBinary(llvm:Builder builder, Scaffold scaffold, bir:DecimalArithmeticBinaryInsn insn) {
+    llvm:Value lhs = buildDecimal(builder, scaffold, insn.operands[0]);
+    llvm:Value rhs = buildDecimal(builder, scaffold, insn.operands[1]);
+    llvm:Value resultWithErr = <llvm:Value>builder.call(scaffold.getRuntimeFunctionDecl(decimalArithmeticFuncs.get(insn.op)), [lhs, rhs]);
+    llvm:BasicBlock continueBlock = scaffold.addBasicBlock();
+    llvm:BasicBlock errBlock = scaffold.addBasicBlock();
+    llvm:Value panicCode = builder.extractValue(resultWithErr, 1);
+    builder.condBr(builder.iCmp("ugt", panicCode, llvm:constInt("i64", 0)), errBlock, continueBlock);
```

Comment:
Can't we branch on the panic code it self?
```suggestion
    builder.condBr(panicCode, errBlock, continueBlock);
```

Reply:
panicCode isn't an i1. Doesn't condBr need an i1?